### PR TITLE
WIP: Update instructions

### PR DESCRIPTION
**Pull request by chronvas**
Third party libraries for Android:
Third party library usage should be encouraged, up to a point, which should be on candidate's discretion to figure out. 
Third party library usage without understanding its effects should also be discouraged, since some can obfuscate a candidate's knowledge on key parts that we aim to figure out the candidate's knowledge on. We should be able to see the candidates’ skills, understanding and knowledge on architecture and Android framework knowledge as well as in testing and even library licensing. If a candidate chooses responsibly to include a third-party library while being fully aware of its effects, its pros and cons, this should be encouraged.

"Kotlin is preferred but Java is also acceptable":
It's clear that Android is moving to using primarily Kotlin, in a way that clearly favours Kotlin over Java, but without blocking Java usage. The instructions should align with that.

### CODE REVIEW

**Comment by rabross** [comment]
Who needs to approve this?

I never did like the blanket _no third-party_ rule and ignore it when reviewing tech tests.
We can either reword it to allow common third party libs (e.g. retrofit, glide, rx, etc) or leave it to the candidates discretion as to how they want to display their ability.
Obviously we want to avoid large app framework libraries because we want to see the candidates architecture and Android framework knowledge.

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Ideally, the user should also be able to trigger a reload of the data from the s
 ### Languages
 For iOS roles, you can write the app in either Swift or Objective-C, but _you should avoid third party libraries_. If you use them, please explain why. For clarity, anything in the Apple SDKs we would consider first-party and an essential part of most any iOS app.
 
-For Android roles, you can write the app in either Kotlin or Java, but _you should avoid third party libraries_.  If you use them, please explain why. For clarity, anything in the Android SDKs we would consider first-party and an essential part of most any Android app.
+For Android roles, Kotlin is preferred but Java is also acceptable. If you use third party libraries, please explain why. Avoid using large app framework libraries, since with this test, your skills on architecture and Android framework knowledge should be visible.
```
Comment (rabross) [inline README.md]: How about:

_Avoid using large, app framework and architecture type libraries as this can obscure from us your understanding of the Android framework as well as good architecture and practices._

**Review by trimbm01 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -38,24 +36,28 @@ Event specific query parameters should be appended to the URL as follows:
...
-Ideally, the user should also be able to trigger a reload of the data from the server.
+### Pull To Refresh
+Ideally, the user should also be able to trigger a reload of the data from the server, using a pull to refresh action.
 
 ### Languages
-For iOS roles, you can write the app in either Swift or Objective-C, but _you should avoid third party libraries_. If you use them, please explain why. For clarity, anything in the Apple SDKs we would consider first-party and an essential part of most any iOS app.
+For iOS roles, you can write the app in either Swift or Objective-C.
 
-For Android roles, you can write the app in either Kotlin or Java, but _you should avoid third party libraries_.  If you use them, please explain why. For clarity, anything in the Android SDKs we would consider first-party and an essential part of most any Android app.
+For Android roles, Kotlin is preferred but Java is also acceptable.
```
Comment (cdoygunbbc) [inline README.md]: I would suggest we should remove Java - as it's less relevant these days. Kotlin is a must for new developers, would we not agree?

Comment (chronvas) [reply README.md]: https://github.com/bbc/news-apps-coding-challenge/pull/1/commits/a227e63a1c300a1a3f3ce72e02fc345603485e72

**Review by cdoygunbbc — APPROVED** [review]
